src/main/kotlin/com/criptext/mail/scenes/emaildetail/EmailDetailScene.kt

```diff
@@ -10,6 +11,7 @@ import android.widget.Toast
 import com.criptext.mail.IHostActivity
 import com.criptext.mail.R
 import com.criptext.mail.api.models.UntrustedDeviceInfo
+import com.criptext.mail.db.KeyValueStorage
```

Comment:
There is no need for this here, the controller already has a reference of the storage, keep the scenes and holders free of that logic, if you need something to be done in the shared prefs storage let the controller know so it can handle what you want.